src/utils/ucc_component.c

```diff
@@ -36,24 +36,17 @@ static ucc_status_t ucc_component_load_one(const char *so_path,
 
     handle = dlopen(so_path, RTLD_LAZY);
     if (!handle) {
-        //TODO use ucc_error
-        fprintf(stderr,
-                "Failed to load UCC component library: %s. "
-                "Check UCC_COMPONENT_PATH or LD_LIBRARY_PATH\n",
-                so_path);
+        ucc_debug("Failed to load UCC component library: %s", so_path);
         goto error;
     }
     iface = (ucc_component_iface_t *)dlsym(handle, iface_struct);
     if ((error = dlerror()) != NULL) {
-        //TODO ucc_error
-        fprintf(stderr, "Failed to get iface %s from %s object\n", iface_struct,
-                so_path);
+        ucc_error("Failed to get iface %s from %s object", iface_struct,
```

Comment:
Let's start with small letters. "Failed" -> "failed"

Reply:
agree, missed this one.